src/SecretsApi/SecretsApi.php

```diff
@@ -34,8 +34,7 @@ private function getBaseURI()
         }
         // If host is still not set, use the default host.
         if (!$host) {
-            // @todo Change to pantheonapi.svc.pantheon.io once alias is set.
-            $host = 'pantheonapi.production.general-01.us-central1.internal.k8s.pantheon.io';
+            $host = 'pantheonapi.pantheon.io';
```

Comment:
This should be [https://api.pantheon.io/customer-secrets/v1](https://api.pantheon.io/customer-secrets/v1) now.